src/main/java/org/takes/rs/RsWithBody.java

```diff
@@ -51,7 +52,8 @@
 public final class RsWithBody extends RsWrap {
 
     /**
-     * Ctor.
+     * Constructs a {@code RsWithBody} with the specified body that will be
+     * encoded into UTF-8 by default.
```

Comment:
@essobedo here we state that `UTF-8` is used by default. What about the rest of constructors? Javadocs are not consistent.